material-ui/package.json

```
@@ -0,0 +1,31 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "remix build",
+    "dev": "remix dev",
+    "postinstall": "remix setup node",
+    "start": "remix-serve build"
+  },
+  "dependencies": {
+    "@emotion/cache": "^11.10.3",
+    "@emotion/react": "^11.10.4",
+    "@emotion/server": "^11.10.0",
+    "@emotion/styled": "^11.10.4",
+    "@mui/material": "5.10.6",
+    "@remix-run/node": "^1.7.1",
+    "@remix-run/react": "^1.7.1",
+    "@remix-run/serve": "^1.7.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "@remix-run/dev": "^1.7.1",
```

Please also add `eslintrc.js` file in the root of the example

```suggestion
    "@remix-run/dev": "*",
    "@remix-run/eslint-config": "*",
```